mroderick/plete
# Add docs from plete-docs

This is far from perfect, but not more broken than it used to be

#### How to verify

1. Check out this branch
1. `npm ci`
1 `npm run start-demo`
1. Navigate to http://localhost:8080/
1. Observe that the demo page loads
1. Stop the server
1. `npm start`
1. Navigate to http://localhost:8080/dev.html
1. Observe that the development page loads
